### ISSUE TITLE
Design System: Update Toggle Component to Avoid aXe Contrast False Positive

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#9943]
## Context

While adding some no violation checks to karma, the document tab flagged a bad color contrast for the active radio option in the toggle button: 
![Screen Shot 2021-12-08 at 10.44.37 AM.png](https://images.zenhubusercontent.com/5e9dcd9a9ff5dfdaffcdf132/edbd1fc6-8ee0-4307-9187-cd4f0b66e330)

It's flagging  as `color contrast of 1.59 (foreground color: #131516, background color: #373a3b,` the[ real contrast is 7.02](https://webaim.org/resources/contrastchecker/?fcolor=C8CBCC&bcolor=373A3B)
![Screen Shot 2021-12-08 at 10.46.47 AM.png](https://images.zenhubusercontent.com/5e9dcd9a9ff5dfdaffcdf132/26a4ef26-1ebe-485b-8635-a74531464f18)

So the ask here is to take a look at the toggle button in the design system to adjust is and get the real color contrast to be picked up in aXe so we can enable this check on the entire document tab and avoid other false positives where this component is used.

You can check this by enabling the disabled test in components/inspector/karma that correlates to this ticket number.
